docs/tutorial/rust/src/getting_started.md

```diff
@@ -12,15 +12,10 @@ cargo new memory
 cd memory
 ```
 
-Then we edit `Cargo.toml` to add the slint dependency:
+Then we edit `Cargo.toml` to add the slint dependency using `cargo add`:
 
-```toml
-[package]
-#...
-edition = "2021"
-
-[dependencies]
-slint = "0.3.4"
+```sh
+cargo add slint@0.3.4
```

Comment:
```suggestion
cargo add slint@1.0.0
```

Since you added the command to change in the CI, and that the version is already 1.0.0 in master


That said, I wonder if we should not just ignore the version here.  Since semver will anyway work.